### DISCUSSION TITLE
Inetrnal editor not working for me

### DISCUSSION THREAD

**Discussion by Comandante1977** [outline/outline#3677] [Self Hosting] [answered]
Can create new doc, edit title and no more.
![image](https://user-images.githubusercontent.com/45628795/174671515-4a824201-eb96-4605-a760-3c6822f9ffb9.png)

**Comment by Comandante1977** [comment, marked as answer]
Solved. It was my mistake. I have nginx as reverse proxy, so these directives are **required**!

    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection "Upgrade";
    proxy_set_header Host $host;

    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;proxy_set_header Host $host;
    proxy_set_header Host $http_host;
    proxy_set_header X-Real-IP $remote_addr;
    proxy_set_header X-Scheme $scheme;
    proxy_set_header X-Forwarded-Proto $scheme;
    proxy_redirect off;

**Comment by lakshmibhargavaBo-Kore** [comment]
Hi @Comandante1977 

I have validated my nginx configuration as mentioned in your answer it's the same but I am still encountering the issues with editor while editing a new document.

Are there any dependencies that nginx.conf has ? or any other suggestions.

**Reply by Comandante1977** [reply]
These directives are only for reverse proxy configuration.
Unfortunately, I've dont work on this project already, so cant see other nginx confs.
Look at this discussion as well https://github.com/chsasank/outline-wiki-docker-compose/issues/1
